src/js/components/visualization/Visualization.jsx

```diff
@@ -93,6 +93,68 @@ class Visualization extends Component {
       typeGroup
     }
   }
+  
+  renderError (error) {
```

Comment:
We should just be able to pull the error from the props instead of passing it in since we are already pulling visualization off of `this.props`.